Atomic/util.py

```diff
@@ -354,7 +363,9 @@ def skopeo_copy(source, destination, debug=False):
     if destination.startswith("docker-daemon"):
         cmd = cmd + ['--remove-signatures']
     cmd = cmd + [source, destination]
-    return skopeo_subprocess(cmd)
+    return_code = call(cmd)
+    if return_code is not 0:
+        raise Exception("The command '{}' failed".format(" ".join(cmd)))
```

Comment:
We can.  I was using subp for cases where no input would be required.  I am happy to alter however.